modules/microshift-config-parameters-table.adoc

```diff
@@ -63,7 +63,7 @@ The following table explains {microshift-short} configuration YAML parameters an
 
 |`tls.minVersion`
 |`VersionTLS12` or `VersionTLS13`
-|Specifies the minimum version of TLS to serve from the API serve. Default is value is `VersionTLS12`. TLS 1.3 ciphers are preset and not configurable.
+|Specifies the minimum version of TLS to serve from the API server. Default is value is `VersionTLS12`. TLS 1.3 ciphers are preset and not configurable.
```

Comment:
```suggestion
|Specifies the minimum version of TLS to serve from the API server. Default value is `VersionTLS12`. TLS 1.3 ciphers are preset and not configurable.
```

Reply:
"TLS 1.3 ciphers are preset and not configurable". This is a confusing statement, the actual meaning is not clear?